src/patient-registration/patient-registration.component.tsx

```diff
@@ -329,6 +343,10 @@ export const PatientRegistration: React.FC = () => {
     return null;
   };
 
+  const cancelRegistration = () => {
+    navigate({ to: '/openmrs/spa/home' });
```

Comment:
Do we? I'd say we navigate to the base path. It should decide what the default route. The base path should not be hardcoded though - we can get it from the global variables; it's `spaBase`.

Reply:
When I try this `navigate({ to: windows.spaBase})`, it takes me to `https://openmrs-spa.org/openmrs/spa` and shows me a blank page.

Reply:
Sorry if I wasn't clear. The base path is the "/openmrs/spa" part. This one can be configured and you can access the latest one in `window.spaBase`. Thus if you want to navigate to `/openmrs/spa/home` you'd navigate to `${window.spaBase}/home`instead - as this will always take you to the right location. Actually you even drop that template string and make it `"${openmrsSpaBase}/home"` - this will be interpolated for you.

Your path is different (as you can see from the URL).